frontend/ui/widget/inputdialog.lua

```diff
@@ -466,7 +467,7 @@ function InputDialog:init()
 end
 
 function InputDialog:onTap()
-    if self.fullscreen or self.add_nav_bar then
+    if self.deny_keyboard_hiding or self.fullscreen or self.add_nav_bar then
```

Comment:
To avoid having to do 3 checks here, you could have in InputDialog:init():
```lua
if self.fullscreen or self.add_nav_bar then
    self.deny_keyboard_hiding = true
end
```